Description: explicit call executable in shebang
 Omit python3: Handled by packaging routines
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2020-10-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/compiler.js
+++ b/src/compiler.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 /**
  * @license
  * Copyright 2010 The Emscripten Authors
--- a/tools/preprocessor.js
+++ b/tools/preprocessor.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 // Copyright 2018 The Emscripten Authors.  All rights reserved.
 // Emscripten is available under two separate licenses, the MIT license and the
 // University of Illinois/NCSA Open Source License.  Both these licenses can be
--- a/third_party/websockify/other/js/websockify.js
+++ b/third_party/websockify/other/js/websockify.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 
 // A WebSocket to TCP socket proxy
 // Copyright 2012 Joel Martin
--- a/third_party/websockify/other/launch.sh
+++ b/third_party/websockify/other/launch.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 
 usage() {
     if [ "$*" ]; then
--- a/third_party/websockify/other/websockify.rb
+++ b/third_party/websockify/other/websockify.rb
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/bin/ruby
 
 # A WebSocket to TCP socket proxy
 # Copyright 2011 Joel Martin
--- a/third_party/websockify/other/wswrap
+++ b/third_party/websockify/other/wswrap
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 
 usage() {
     echo "Usage: $(basename $0) PORT CMDLINE"
--- a/tests/sourcemap2json.js
+++ b/tests/sourcemap2json.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 // Copyright 2013 The Emscripten Authors.  All rights reserved.
 // Emscripten is available under two separate licenses, the MIT license and the
 // University of Illinois/NCSA Open Source License.  Both these licenses can be
--- a/third_party/websockify/rebind
+++ b/third_party/websockify/rebind
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 
 usage() {
     echo "Usage: $(basename $0) OLD_PORT NEW_PORT COMMAND_LINE"
--- a/tools/acorn-optimizer.js
+++ b/tools/acorn-optimizer.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 
 'use strict';
 
--- a/tools/lz4-compress.js
+++ b/tools/lz4-compress.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 // Copyright 2015 The Emscripten Authors.  All rights reserved.
 // Emscripten is available under two separate licenses, the MIT license and the
 // University of Illinois/NCSA Open Source License.  Both these licenses can be
